### PR TITLE
Refresh of Pod Security Admission.

### PR DESCRIPTION
**Pull request by jhkrug**
There are some questions in TODOs.

Thanks.

### CODE REVIEW

File: docs/tasksDir/pod-security-admission-with-kubewarden.md [hunk 1]
```diff
@@ -1,50 +1,44 @@
...
 
-In addition to choosing the profile to be used, the Kubernetes operators can decide
-what kind of action is going to be performed by Pod Security Admission controller
-once a violation is detected. The possible actions are: `enforce`, `audit` and
-`warn`.
+A PSA controller can perform actions when a violation is detected.
+The actions are: `enforce`, `audit`, and `warn` and can be configured.
 
-Pod Security Admission is not a full replacement for the old Pod Security Policies.
+<!--TODO:Still suffering these limitations? Next three paras may need update.-->
```
Comment (viccuad) [inline docs/tasksDir/pod-security-admission-with-kubewarden.md:21]: Yes, those limitations may be there for a long time. We don't need to write this in the docs, but PSAs are minimal and limited in scope by design; the Kubernetes team doesn't want to add a sprawling API without direction. The idea of the policies are securities working group is to let a standard emerge from the community first (e.g: Kubewarden), then think about adding those functionalities to the Kubernetes API.

Maybe we could reword it to:

> At the time of writing, with Kubernetes 1.26, it suffers the following limitations:

File: docs/tasksDir/pod-security-admission-with-kubewarden.md [hunk 2]
```diff
@@ -150,20 +155,38 @@ spec:
...
+
+<!--
+
+Current sentence is
+
+Now PSA allows the pod to be created, but the Pod creation will still fail.
+
+Is this better?
+
+Now the PSA process passes pod creation but the Kubewarden process of pod creation fails.
```
Comment (viccuad) [inline docs/tasksDir/pod-security-admission-with-kubewarden.md:170]: that's incorrect, as I understand it, Kubewarden is not yet in the mix.
I think we are just showcasing a shortcoming of PSA behaviour, that then we fix with a Kubewarden mutating policy. So I suppose the current phrase is okeish.

**Review by viccuad — APPROVED** [review]
LGTM!
This is a good refresh! More concise but better explained, plus style changes too, thanks!
Added some clarifications and nits.

**Review by jvanz — APPROVED** [review]

**Comment by jvanz** [comment]
LGTM. Thanks for the enhancements and I've agree with the Victor's feedback.

**Comment by jhkrug** [comment]
Thanks, will follow up on Monday.

File: docs/tasksDir/pod-security-admission-with-kubewarden.md (hunk 1 above)
Comment (flavio) [reply docs/tasksDir/pod-security-admission-with-kubewarden.md:21]: It applies also to 1.28

File: docs/tasksDir/pod-security-admission-with-kubewarden.md [hunk 3]
```diff
@@ -113,7 +114,11 @@ kubectl get deploy -n my-namespace nginx-deployment -o json | jq ".status.condit
...
 ```
 
-The user can fix this issue removing the `runAsUser` from the container definition:
+You can fix this by removing the `runAsUser: 0` from the container definition:
+
+<details>
+
+<summary><code>kubectl</code> command configuring a resource without <code>runAsUser: 0</code></summary>
 
 ```console
```
Comment (jvanz) [inline docs/tasksDir/pod-security-admission-with-kubewarden.md:123]: This is not a requirement to merge this PR. But I think we should use `shell` syntax as the other code blocks. It's visually more pleasant and easy to read.

**Review by jvanz — APPROVED** [review]

File: docs/tasksDir/pod-security-admission-with-kubewarden.md (hunk 1 above)
Comment (jhkrug) [reply docs/tasksDir/pod-security-admission-with-kubewarden.md]: Thanks.

File: docs/tasksDir/pod-security-admission-with-kubewarden.md (hunk 2 above)
Comment (jhkrug) [reply docs/tasksDir/pod-security-admission-with-kubewarden.md]: Ok, think I have it now.

File: docs/tasksDir/pod-security-admission-with-kubewarden.md [hunk 4]
```diff
@@ -5,38 +5,38 @@ description: Using Pod Security Admission with Kubewarden, since the Kubernetes
...
 
 PSA simplifies securing the Pods in Kubernetes clusters.
 
-PSA has three profiles for namespaces: `privileged`, `baseline`, and `restricted`.
+PSA has three profiles (described [in Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/)) for namespaces:
 
-The `privileged` profile has few limitations.
-The `restricted` profile tightens Pod permissions.
+- **privileged**, providing the widest range of permissions
+- **baseline**, to prevent now privilege escalations
```
Comment (flavio) [inline docs/tasksDir/pod-security-admission-with-kubewarden.md:16]:
```suggestion
- **baseline**, to prevent new privilege escalations
```

File: docs/tasksDir/pod-security-admission-with-kubewarden.md [hunk 5]
```diff
@@ -86,6 +85,7 @@ spec:
         image: nginx:1.14.2
         securityContext:
           runAsNonRoot: true
+// highlight-next-line
```
Comment (flavio) [inline docs/tasksDir/pod-security-admission-with-kubewarden.md:88]: cool, I didn't know about that!

**Review by flavio — APPROVED** [review]
LGTM, I left a minor fix
